src/openzaak/components/catalogi/tests/admin/test_admin_publish.py

```diff
@@ -99,10 +99,11 @@ def test_publish_zaaktype(self, m, mock_notif):
         )
         self.assertIsNone(publish_button)
 
-        form = response.forms["zaaktype_form"]
+        url = reverse("admin:catalogi_zaaktype_publish", args=(zaaktype.pk,))
+        publish_page = self.app.get(url)
 
         with self.captureOnCommitCallbacks(execute=True):
-            response = form.submit("_publish").follow()
+            response = publish_page.form.submit("_publish").follow()
 
         zaaktype.refresh_from_db()
         self.assertFalse(zaaktype.concept)
```

Comment:
The notification bellow now fails to be called and I cant find why. @annashamray do you know?

Also why cant I comment on unchanged lines?

Reply:
Because Github isn't Bitbucket, not being able to comment on unchanged lines in a PR is one of my (few) annoyances with Github

Reply:
This actually seems to be from the SideEffectsMixin